hoomd/write/gsd.py

```diff
@@ -212,20 +234,56 @@ def write(state, filename, filter=All(), mode='wb', log=None):
         writer = _hoomd.GSDDumpWriter(state._cpp_sys_def, Periodic(1), filename,
                                       state._get_group(filter), mode, False)
 
-        if log is not None:
-            writer.log_writer = _GSDLogWriter(log)
+        if all((logger is not None, log is not None)):
+            warnings.warn(
+                "log and logger keyword arguments passed to write.GSD.write()."
+                " Keyword argument \"log\" is deprecated since v3.9.0."
+                " Ignoring log and using logger instead.", DeprecationWarning)
+        elif logger is None and log is not None:
+            warnings.warn(
+                "log keyword arguments passed to write.GSD.write() is"
+                " deprecated since v3.9.0. Use logger instead.",
+                DeprecationWarning)
+            logger = log
+
+        if logger is not None:
+            writer.log_writer = _GSDLogWriter(logger)
         writer.analyze(state._simulation.timestep)
 
+    @property
+    def logger(self):
+        """hoomd.logging.Logger: Provide log quantities to write.
+
+        May be `None`.
+        """
+        return self._logger
+
     @property
     def log(self):
         """hoomd.logging.Logger: Provide log quantities to write.
 
         May be `None`.
+
+        .. deprecated:: v3.9.0
+           ``log`` will be renamed to ``logger`` in v4.
         """
```

Comment:
```suggestion
        """
        warnings.warn(
            "log property  is deprecated since v3.9.0. Use logger instead.",
            DeprecationWarning)
```